app/src/main/java/sdp/moneyrun/ui/menu/MenuActivity.java

```diff
@@ -75,6 +75,9 @@ protected void onCreate(Bundle savedInstanceState) {
     public void runFunctionalities(){
         //Setting the current player object
         user = (Player) getIntent().getSerializableExtra("user");
+        if(user == null){
```

Comment:
This should be tested to improve coverage

Reply:
fair enough